### ISSUE TITLE
CryptBinaryToString should be DeclareWide?

### ISSUE THREAD

**Issue by GCuser99** [fafalone/WinDevLib#26]
in wdAPICrypto.twin:
```vba
Public Declare PtrSafe Function CryptBinaryToStringW Lib "crypt32" (pbBinary As Any, ByVal cbBinary As Long, ByVal dwFlags As CRYPT_STRING_OPTIONS, ByVal pszString As LongPtr, pcchString As Long) As BOOL
Public Declare PtrSafe Function CryptBinaryToString Lib "crypt32" Alias "CryptBinaryToStringW" (pbBinary As Any, ByVal cbBinary As Long, ByVal dwFlags As CRYPT_STRING_OPTIONS, ByVal pszString As String, pcchString As Long) As BOOL
```

Thx!

**Comment by fafalone** [comment]
Yup should be.

Btw, started finding APIs missed by your conversion:

```
Public DeclareWide PtrSafe Function MonitorFromPoint Lib "user32" (ByVal pt As LongLong, ByVal dwFlags As DefaultMonitorValues) As LongPtr
Public DeclareWide PtrSafe Function MonitorFromWindow Lib "user32" (ByVal hwnd As LongPtr, ByVal dwFlags As DefaultMonitorValues) As LongPtr
Public DeclareWide PtrSafe Function MonitorFromRect Lib "user32" (lprc As RECT, ByVal dwFlags As DefaultMonitorValues) As LongPtr
```

No biggie, it got 99% of them, just curious how some escaped. Don't worry about going crazy trying to fix it, I'll just get the stragglers manually as I see them

**Comment by GCuser99** [comment]
Sure Jon - do you want me to use same inputs or do you have an updated set to provide me so I can keep in sync with you?

**Comment by fafalone** [comment]
Thanks a bunch. Here's the latest version of the files to run. No releases lately because I got bogged down in 'I should add this, and that, and that! So there's a bunch of unfinished stuff, and no major bugs.  

[Sources.zip](https://github.com/fafalone/WinDevLib/files/15278239/Sources.zip)

**Comment by GCuser99** [comment]
Same format as before...

[wdl 240510.zip](https://github.com/fafalone/WinDevLib/files/15278685/wdl.240510.zip)

**Comment by fafalone** [comment]
Thanks looks great!

**Comment by fafalone** [comment]
Fixed in v8.0.402, now available. 

Thanks for reporting and thanks again for the help with the APIs. I added a thanks at the bottom of the API section of the readme on the front page of the repo.
